docker-compose/travel-agency-services-macos-windows.yml

```diff
@@ -0,0 +1,40 @@
+version: '2'
```

Comment:
@cristianonicolai finally I achieve to try this on windows. To make it run I had to change this version to 2.1 having the same issue than is explained here. 
https://github.com/docker/compose/issues/4106#issuecomment-258590985
It's just a comment, maybe is related with the environment where I'm running, but I just installed the last version of docker-desktop for windows.

Reply:
@nmirasch thanks, I've updated the version to 2.1.
@DuncanDoyle do you mind checking again if this change doesn't break for you?